src/pages/styles/Paper.scss

```diff
@@ -148,4 +149,46 @@
     font-size: 16px;
     color: darkgray;
   }
+
+  .card {
+    border: none;
+    background: lighten($gray-200, 1.5);
+    border-radius: 8px;
+
+    header {
+      color: white;
+      font-weight: bold;
+      padding: 10px;
+      border-radius: 8px;
+      margin-bottom: 20px;
+      background: $primary;
+      @include font-size(1.2rem);
+    }
+
+    // Card subheader
+    .subheading {
+      border-bottom: 1px solid $gray-300;
+      font-style: italic;
+    }
+
+    .col {
+      margin-left: 10px
+    }
+
+    .hepa-result {
```

Comment:
I wonder if it'd be wise to start moving CSS for specific blog posts to their own files?

Reply:
Maybe? My suspicion is that we'll want most of them to share most of the same styles.